index.html

```diff
@@ -6,6 +6,19 @@
         <title>Javiasy - API Project</title>
     </head>
     <body>
+        <div id="bookWrapper">
+            <ul id="bookList">
+                <li bookId="1">A Game of Thrones</li>
+                <li bookId="2">A Clash of Kings</li>
+                <li bookId="3">A Storm of Swords</li>
+                <li bookId="4">A Feast For Crows</li>
+                <li bookId="5">A Dance with Dragons</li>
+            </ul>
+        </div>
+        <div id="characterWrapper">
+            <ul id="characterList">
 
```

Comment:
Might be better not to leave the space between the opening and closing <ul> tags, since a test will read the space / return character - if we're testing to see whether the <ul> is empty, it will return false at the moment. I think.